libs/components/indicators/src/lib/modules/tokens/token.component.ts

```diff
@@ -36,14 +36,12 @@ export class SkyTokenComponent {
    * @default "Remove item"
    */
   @Input()
-  public set ariaLabel(value: string) {
-    this._ariaLabel = value;
+  public set ariaLabel(value: string | undefined) {
+    this.#ariaLabelOrDefault = value || this.#getDefaultAriaLabel();
```

Comment:
If they pass `''` I'm assuming we want to revert to the default label? If not, we can use null coalescing to only set the default when `undefined` is passed.

Reply:
I could go either way. I can't really see a situation where the consumer _wouldn't_ want an ARIA label, so setting it to `''` should probably set it back to the default.